### ISSUE TITLE
Resource Usage Monitoring

### ISSUE THREAD

**Issue by brentstineman** [SimpleCloudManagerProject/SCAMP#150]
A placeholder for discussions and questions related to SCAMP monitoring of resource consumption.

**Comment by jaroklc** [comment]
>Backlog/JumpIn: SCAMP v1.0 will not track if the total allocation to all users exceeds the budget allocation for the group. This would need to be checked as users are added to the group or user budget allocations are changed.

This might create a situation when a group admin sets a large individual allocation (i.e. by mistake) to a group member and the member burns the whole budget for the group. I would vote for a solution where the group admin is only able to set default member quota values and individual quotas with the following constraint: the sum of individual group member's allocations is less that the group's budget.

**Comment by GabrieleCastellani** [comment]
I agree with the proposal of "the sum of individual group member's allocations is less that the group's budget."

**Comment by GabrieleCastellani** [comment]
>Backlog/JumpIn: SCAMP v1.0 will not track if the total allocation to all users exceeds the budget allocation for the group. This would need to be checked as users are added to the group or user budget allocations are changed.

I think we can overcome this limitation with the actual design

**Comment by brentstineman** [comment]
Just created a backlog task for this one. :) See reference above.
